docs/book/src/developer/tilt.md

```diff
@@ -186,8 +186,18 @@ tilt up
 
 This will open the command-line HUD as well as a web browser interface. You can monitor Tilt's status in either
 location. After a brief amount of time, you should have a running development environment, and you should now be able to
-create a cluster. Please see the [Usage section in the Quick
-Start](https://cluster-api.sigs.k8s.io/user/quick-start.html#usage) for more information on creating workload clusters.
+create a cluster. There are [example worker cluster
```

Comment:
I also had to `kind load docker-image` for each of the controller image.
Can we document that too please?

Reply:
Hmm, I didn't get that from deploying on a fresh system. Do you know the conditions that led to needing to do that? Sounds like maybe that is a different issue.

Reply:
I was using a `kind ` cluster with
```
＄ cat ./tilt-settings.json
{
  "default_registry": "gcr.io/k8s-staging-cluster-api",
  "enable_providers": ["docker", "kubeadm-bootstrap", "kubeadm-control-plane"]
}
```

Did you by any chance push the test images to your local registry? not sure what your container registry setup was.

Reply:
My `tilt-settings.json` file is actually identical to that right now. So I didn't have to perform any extra steps using that config. I did:

```sh
tilt up
s
```

in one screen, then:

```sh
k apply -f test/infrastructure/docker/examples/simple-cluster.yaml
```

After a minute I was able to verify with:

```sh
k get cluster
k get dockermachine
docker ps
```

Everything else looks good.

Reply:
Doing this gives me `ImagePullBackOff` on all the controllers.
IIRC you were using `localhost:5000` for your `default_registry`, no?

Reply:
I had started with that, but just to eliminate any potential issues there I cleaned everything up and started again with the steps above based on Fabrizio's input.

Might be good to file an issue to capture the problems you had and see if anyone else runs in to it. Maybe if we kind find others with the same problem we can figure out what might be common between them to see why it's happening.

Reply:
sounds good.

Reply:
Per offline investigation, it was found that the instructions in the tilt documentation to spin up a `kind` cluster need to be updated to be inline with the ones on the `quick-start` guide. Specifically, share the docker socket mount.
I can do that in a follow-up PR.

Reply:
Actually tilt should push those images into the kind cluster imho.

Reply:
@sbueringer Yes, using the misconfigured kind cluster, all the controllers go into image pull backoffs